pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -33,6 +34,16 @@
 #define ENABLE_DEBUG (0)
 #include <debug.h>
 
+#define TEST_FATFS_MAX_VOL_STR_LEN 14 /* "-2147483648:/\0" */
+
+#ifndef CONFIG_FATFS_TYPE
+#if FF_FS_EXFAT
+#define CONFIG_FATFS_TYPE   FM_EXFAT
+#else
+#define CONFIG_FATFS_TYPE   FM_FAT32
+#endif
+#endif
```

Comment:
This could use some documentation. Does it make sense to move it to `sys/include/fs/fatfs.h` and document the options?

Reply:
Ok, moved it there.